v 1.2.7 npm install - postinstall error

Installing v. 1.2.7 with npm, I get the following error:

```bash
> react-query@1.2.7 postinstall /Users/matteo/Development/react-bricks/node_modules/react-query
> node ./scripts/postinstall.js || exit 0

internal/modules/cjs/loader.js:800
    throw err;
    ^

Error: Cannot find module '/Users/matteo/Development/react-bricks/node_modules/react-query/scripts/postinstall.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

```

--- a/package.json
+++ b/package.json
@@ -30,7 +30,8 @@
   },
   "files": [
     "dist",
-    "types"
+    "types",
+    "scripts"
   ],
   "dependencies": {
     "@scarf/scarf": "^1.0.0",

